packages/common/__tests__/cors.test.ts

```diff
@@ -0,0 +1,82 @@
+import { Request } from 'cross-undici-fetch'
+import { request } from 'http'
+import { createServer } from '../src/server'
+
+describe('CORS', () => {
```

Comment:
@ardatan think we need tests for OPTIONS requests too! Happy to add them, but not sure best way to coordinate?